modules/nextflow/src/main/groovy/nextflow/config/parser/legacy/ConfigBase.groovy

```diff
@@ -14,14 +14,15 @@
  * limitations under the License.
  */
 
-package nextflow.config
+package nextflow.config.parser.legacy
```

Comment:
it would be preferable to not shuffle stuff around and use `Legacy`.

Reply:
I think this is a reasonable change to keep related classes organized. You have done a similar thing many times, e.g. `nextflow.executor.local` or your proposed operator cleanup for the provenance tracking

Reply:
That's fair, what I think it should be avoid the use of the term "legacy" in class or package name, as it's a bad practice to use the term "new" when introducing new class. It would be preferable just to use `v1` for current ones and v2 for new ones.